--- .mkshrc	2026-05-20 11:05:52.364000000 -0600
***************
*** 18,24 ****
  [[ -s /etc/mkshrc ]] && \. /etc/mkshrc
  
  # work around #813164
! \\builtin alias ls='ls -N'
  
  # prepend $debian_chroot support to PS1
  p=$'\001'
--- 18,24 ----
  [[ -s /etc/mkshrc ]] && \. /etc/mkshrc
  
  # work around #813164
! \\builtin alias ls='ls -N --color=auto'
  
  # prepend $debian_chroot support to PS1
  p=$'\001'
***************
*** 50,53 ****
--- 50,78 ----
  #\\builtin export LANG=C.UTF-8 LC_CTYPE=C.UTF-8
  #\\builtin export LC_ALL=C.UTF-8
  
+ \\builtin export HISTFILE="$HOME"/.mksh_history
+ 
+ # ssh-agent
+ if [ -z "$SSH_AUTH_SOCK" ]
+ then
+        \\builtin export SSH_AUTH_SOCK="$HOME"/.ssh/ssh-agent.socket
+        \\builtin export SSH_ASKPASS_REQUIRE="never"
+ fi
+ 
+ "$HOME"/.ssh/add_ssh_keys
+ 
+ # pipwire
+ "$HOME"/local/bin/start_audio
+ 
+ FORTUNE="$(which fortune 2>&1)" >/dev/null
+ if [ -x "$FORTUNE" ]
+ then
+         $FORTUNE
+ fi
+ 
+ if [ "$TERM" = "linux" ]
+ then
+ 	start_tmux
+ fi
+ 
  \: make sure this is the last line, to ensure a good return code
